ddtrace/span.py

```diff
@@ -255,6 +255,9 @@ def pprint(self):
         lines.extend((" ", "%s:%s" % kv) for kv in sorted(self.meta.items()))
         return "\n".join("%10s %s" % l for l in lines)
 
+    def tracer(self):
```

Comment:
do we need to make it "available" as a ``Span`` public API? or do we need it only for internal instrumentation? in that case I suggest to keep it "internal".